.github/workflows/e2eTest.yml

```diff
@@ -22,6 +22,19 @@ jobs:
           java-version: '11'
           distribution: 'corretto'
 
+      - name: Generate Cache Key from Dockerfiles
```

Comment:
> However, it turns out that this key is evaluated at runtime

Good find and workaround, since this is a bug in the action, could you create an issue so they can fix it in a future release?

Reply:
Done: https://github.com/ScribeMD/docker-cache/issues/823

Reply:
(I have most of a PR for them, but the tests are pretty brittle and I'm not sure how much time I want to put into resolving this before I get an indication about whether they'd be open to it anyways)